README.md

```diff
@@ -2,7 +2,6 @@
 
 <div align="center">
   <h3><b>Leaderboard</b></h3>
-
 </div>
 
 # 📗 Table of Contents
```

Comment:
- [Optional] Please kindly update the date in LISENCE.md file

![Screenshot from 2023-06-30 19-41-53](https://github.com/cancelei/Leaderboard/assets/63087491/48e36811-141f-4d93-8e73-1253779786c7)